deploy.sh

```diff
@@ -0,0 +1,3 @@
+docker stack rm groff
+docker-compose pull
+docker stack deploy -c docker-compose.yml 
```

Comment:
No need to do these 3 commands. You can use the `--force` command to update the docker stack with the latest images.

Reply:
Noted, Have made the change